docs/qiita-create-study.rst

```diff
@@ -0,0 +1,164 @@
+Creating a study in Qiita
+=========================
+
+This tutorial will walk you through creation of your account and a test study
+in Qiita.
+
+Getting CMI Workshop example data
+---------------------------------
+
+First, we'll download some `example data <https://www.dropbox.com/s/1vcnjqo6kkmetp6/qiita-files.zip?dl=0>`__. These files contain both 16S and shotgun metagenomics 
+data for 12 samples from the American Gut Project. 
+
+For this tutorial, the relevant files are::
+
+    qiita-files/16S/*.fastq.gz        # The actual 16S sequences, one per sample
+    qiita-files/sample_information.txt   # The sample information file
+    qiita-files/prep_template_16S.txt # The prep information file
+
+
+Next, we'll sign up for Qiita and create a study for these data.
+
+
+Signing up for a Qiita account
+------------------------------
+
+Open your browser (it must be Chrome or Firefox) and go to `Qiita <https://qiita.ucsd.edu>`__ (https://qiita.ucsd.edu).
+
+Click on "New User".
+
+.. figure::  images/image14.png
+   :align:   center
+
+.. figure::  images/image07.png
+   :align:   center
+
+The "New User" link brings you to a page on which you can create a new
+account. Optional fields are indicated explicitly, while all other
+fields are required. Once the form is submitted, an email will be sent
+to you containing instructions on how to verify your email address.
+
+Once you have created your account, you can log into the system by
+entering your email and password.
+
+.. figure::  images/image03.png
+   :align:   center
+
+
+Creating a test study
+---------------------
+
+Studies are the main source of data for Qiita. Studies can contain only one set
+of samples but can contain multiple sets of raw data, each of which can have a
+different preparation -- for example, 16S, shotgun metagenomics, and metabolomics.
```

Comment:
...or even multiple preparations of the same type (e.g., a plate rerun, biological and technical replicates, etc)